functions/Set-DbaBackupPath.ps1

```diff
@@ -0,0 +1,149 @@
+function Set-DbaBackupPath {
+    <#
+        .SYNOPSIS
+            Sets SQL Server default backup directory to a new value then displays information this setting.
+
+        .DESCRIPTION
+            Sets SQL Server default backup directory then displays information relating to the configuration setting.
+
+            If the path does not exist it will try to create it and grant the instance service account access.
+
+        .PARAMETER SqlInstance
+            Allows you to specify a comma separated list of servers to query.
+
+        .PARAMETER Path
+            Specifies the new backup directory
+
+        .PARAMETER SqlCredential
+            Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+
+            $scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.
+
+            Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials
+            being passed as credentials. To connect as a different Windows user, run PowerShell as that user.
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .PARAMETER WhatIf
+            Shows what would happen if the cmdlet runs. The cmdlet is not run.
+
+        .PARAMETER Confirm
+            Prompts you for confirmation before running the cmdlet.
+
+        .NOTES
+            Tags: Storage, DisasterRecovery, Backup
+            Author: Andrew Wickham, @awickham, www.awickham.com
+
+            dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+            Copyright (C) 2016 Chrissy LeMaire
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+        .LINK
+            https://dbatools.io/Set-DbaBackupPath
+
+        .EXAMPLE
+            Set-DbaBackupPath -SqlInstance 'sqlserver1' -Path 'Q:\Backups'
+
+            Set the backup path to Q:\Backups for one server named "sqlserver1"
+
+        .EXAMPLE
+            Get-DbaRegisteredServer -SqlInstance sqlserver | Set-DbaBackupPath 'Q:\Backups'
+
+            Find all servers in SQL Server Central Management server, then pipe those to Set-DbaBackupPath
+            and set the backup directory to Q:\Backups.
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
+    param (
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer", "SqlServers", "ComputerName")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [Alias("Credential")]
+        [PSCredential]$SqlCredential,
```

Comment:
Because you will be creating a path on the local server we will need to include an credential parameter.